src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -80,25 +77,7 @@ class SourcesTree extends Component<Props, State> {
 
   constructor(props) {
     super(props);
-    this.state = createTree(
-      this.props.sources,
-      this.props.debuggeeUrl,
-      this.props.projectRoot
-    );
-    this.focusItem = this.focusItem.bind(this);
-    this.selectItem = this.selectItem.bind(this);
-    this.getPath = this.getPath.bind(this);
-    this.getIcon = this.getIcon.bind(this);
-    this.onContextMenu = this.onContextMenu.bind(this);
-    this.renderItem = this.renderItem.bind(this);
-
-    this.queueUpdate = throttle(function() {
-      if (!this.mounted) {
-        return;
-      }
-
-      this.forceUpdate();
-    }, 50);
+    this.state = createTree(this.props);
```

Comment:
Think this should be more verbose, spreading the parameters for easier reading instead of putting the whole object in there.